src/App.tsx

```diff
@@ -1,24 +1,140 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import * as todosService from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Errors } from './types/Errors';
+import { FilterValue } from './types/FilterValue';
+import { ErrorNotification } from './components/ErrorNotification';
+// import {handleError, setErrorMessage} from "./helpers";
 
-const USER_ID = 0;
+const USER_ID = 12037;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState(FilterValue.All);
+  const [tempTodo, setTempTodo] = useState<Omit<Todo, 'id'> | null>(null);
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.Null);
+  const [showLoader, setShowLoader] = useState(false);
+
+  useEffect(
+    () => {
+      todosService.getTodos(USER_ID)
+        .then(setTodos)
+        .catch(() => setErrorMessage(Errors.LoadTodos))
+        .finally(() => setTimeout(() => setErrorMessage(Errors.Null), 3000));
```

Comment:
- in the positive case you don't have an error, so you don't need to set timeout.
- you do the same when you call other APIs, let's create a simple handler to do it and then you can just pass the error message in all catch blocks
```typescript
const handleError = (errorMessage: string) => {
  setErrorMessage(errorMessage);
  setTimeout(() => setErrorMessage(Errors.Null), 3000)
};
```